<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Koren A. Nishina - Publications</title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
</head>

<body>
    <nav class="navbar navbar-default" style="background-color: white">
        <div class="container-fluid">
            <ul class="nav navbar-nav">
                <li> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </li>
                <li><img src="Images/icon.jpeg" width="90"></li>
                <li> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </li>
                <li><a href="index.html"><strong>Home</strong></a></li>
                <li><a href="personal.html"><strong>About</strong></a></li>
                <li><a href="projects.html"><strong>Projects</strong></a></li>
                <li><a href="papers.html"><strong>Publications</strong></a></li>
                <li><a href="peruse.html"><strong>Ongoing</strong></a></li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
                <li><a data-toggle="tooltip" data-placement="top" title="GitHub" href="https://github.com/knishina/" target="_blank"><img src="Images/github.png" alt="github" width="28"></a></li>
                <li><a data-toggle="tooltip" data-placement="top" title="LinkedIn" href="https://linkedin.com/in/knishina/" target="_blank"><img src="Images/linkedin.png" alt="linkedin" width="25"></a></li>
                <li><a data-toggle="tooltip" data-placement="top" title="Gmail"  href="mailto:koren.a.nishina@gmail.com"><img src="Images/gmail.jpeg" alt="gmail" width="28"></a></li>
                <li> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </li>
            </ul>
        </div>
    </nav>
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <br />
        <h2 style="margin-left: 50px;"><strong>Publications</strong></h2>
        <br />
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/PLOSP.png" alt="PLOS Pathogens">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1005017" target="_blank">A Structural and Functional Comparison Between Infectious and Non-Infectious Autocatalytic Recombinant PrP Conformers</a></h2>
            <h4>Noble GP, Wang DW, Walsh DJ, Barone JR, Miller MB, Nishina KA, Li S, Supattapone S</h4>
            <h4>PLoS Pathog. 2015 Jun 30;11(6):e1005017</h4>
            <p align="justify">Infectious prions contain a self-propagating, misfolded conformer of the prion protein termed PrP<sup>Sc</sup>. A critical prediction of the protein-only hypothesis is that autocatalytic PrP<sup>Sc</sup> molecules should be infectious. However, some autocatalytic recombinant PrP<sup>Sc</sup> molecules have low or undetectable levels of specific infectivity in bioassays, and the essential determinants of recombinant prion infectivity remain obscure. To identify structural and functional features specifically associated with infectivity, we compared the properties of two autocatalytic recombinant PrP conformers derived from the same original template, which differ by >105-fold in specific infectivity for wild-type mice. Structurally, hydrogen/deuterium exchange mass spectrometry (DXMS) studies revealed that solvent accessibility profiles of infectious and non-infectious autocatalytic recombinant PrP conformers are remarkably similar throughout their protease-resistant cores, except for two domains encompassing residues 91-115 and 144-163. Raman spectroscopy and immunoprecipitation studies confirm that these domains adopt distinct conformations within infectious versus non-infectious autocatalytic recombinant PrP conformers. Functionally, <i>in vitro</i> prion propagation experiments show that the non-infectious conformer is unable to seed mouse PrP<sup>C</sup> substrates containing a glycosylphosphatidylinositol (GPI) anchor, including native PrP<sup>C</sup>. Taken together, these results indicate that having a conformation that can be specifically adopted by post-translationally modified PrP<sup>C</sup> molecules is an essential determinant of biological infectivity for recombinant prions, and suggest that this ability is associated with discrete features of PrP<sup>Sc</sup> structure.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
        
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/JVI.png" alt="Journal of Virology" width="230">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://jb.asm.org/content/190/6/2239.long" target="_blank">Prion protein glycosylation is not required for strain-specific neurotropism</a></h2>
            <h4>Piro JR, Harris BT, Nishina K, Soto C, Morales R, Rees JR, Supattapone S</h4>
            <h4>J Virol. 2009 Jun;83(11):5321-8</h4>
            <p align="justify">In this study, we tested the hypothesis that the glycosylation of the pathogenic isoform of the prion protein (PrP<sup>Sc</sup>) might encode the selective neurotropism of prion strains. We prepared unglycosylated cellular prion protein (PrP<sup>C</sup>) substrate molecules from normal mouse brain by treatment with PNGase F and used reconstituted serial protein cyclic misfolding amplification reactions to produce RML and 301C mouse prions containing unglycosylated PrP<sup>Sc</sup> molecules. Both RML- and 301C-derived prions containing unglycosylated PrP<sup>Sc</sup> molecules were infectious to wild-type mice, and neuropathological analysis showed that mice inoculated with these samples maintained strain-specific patterns of PrP<sup>Sc</sup> deposition and neuronal vacuolation. These results show that PrP<sup>Sc</sup> glycosylation is not necessary for strain-dependent prion neurotropism.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>

    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/JB.png" alt="Journal of Bacteriology" width="230">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://jb.asm.org/content/190/6/2239.long" target="_blank">SarA of <i>Staphylococcus aureus</i> binds to the sarA promoter to regulate gene expression</a></h2>
            <h4>Cheung AL, Nishina K, Manna AC</h4>
            <h4>J Bacteriol. 2008 Mar;190(6):2239-43</h4>
            <p align="justify">The 375-bp sarA open reading frame is driven by three promoters, P1, P3, and P2. Using gel shift and DNase I footprinting assays, we found that SarA binds to two 26-bp sequences and one 31-bp sequence within the P1 and P3 promoters, respectively. Together with the results of transcription analyses, our data indicate that SarA binds to its own promoter to down-regulate sarA expression.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
                
    <div class="row col-md-12 col-sm-12 col-xs-12">
       <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/IJBCB.png" alt="International Journal of Biochemistry and Cell Biology" width="220">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2274939/" target="_blank">The SarA protein family of <i>Staphylococcus aureus</i></a></h2>
            <h4>Cheung AL, Nishina KA, Trotonda MP, Tamber S</h4>
            <h4>Int J Biochem Cell Biol. 2008;40(3):355-61</h4>
            <p align="justify"><i>Staphylococcus aureus</i> is widely appreciated as an opportunistic pathogen, primarily in hospital-related infections. However, recent reports indicate that <i>S. aureus</i> infections can now occur in other wise healthy individuals in the community setting. The success of this organism can be attributed to the large array of regulatory proteins, including the SarA protein family, used to respond to changing microenvironments. Sequence alignment and structural data reveal that the SarA protein family can be divided into three subfamilies: 1) single domain proteins; 2) double domain proteins and 3) MarR homologs. Structural studies have also demonstrated that SarA, SarR, SarS, MgrA, and thus possibly all members of this protein family are winged helix proteins with minor variations. Mutagenesis studies of SarA disclose that the winged helix motifs are important for DNA binding and function. Recent progress concerning the functions and plausible mechanisms of regulation of SarA and its homologs are discussed.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>

    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/AMJP.jpeg" alt="American Journal of Pathology" width="220">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://ajp.amjpathol.org/article/S0002-9440(10)62384-X/fulltext" target="_blank">Minocycline Modulates Neuroinflammation Independently of Its Antimicrobial Activity in <i>Staphylococcus aureus</i>-Induced Brain Abscess</a></h2>
            <h4>Tammy Kielian, Nilufer Esen, Shuliang Liu, Nirmal K. Phulwani, Mohsin M. Syed, Napoleon Phillips, Koren Nishina†, Ambrose L. Cheung, Joseph D. Schwartzman, Jorg J. Ruhe</h4>
            <h4>Am J Pathol. 2007 Oct;171(4):1199-214</h4>
            <p align="justify">Minocycline exerts beneficial immune modulatory effects in several noninfectious neurodegenerative disease models; however, its potential to influence the host immune response during central nervous system bacterial infections, such as brain abscess, has not yet been investigated. Using a minocycline-resistant strain of <i>Staphylococcus aureus</i> to dissect the antibiotic's bacteriostatic versus immune modulatory effects in a mouse experimental brain abscess model, we found that minocycline significantly reduced mortality rates within the first 24 hours following bacterial exposure. This protection was associated with a transient decrease in the expression of several proinflammatory mediators, including interleukin-1beta and CCL2 (MCP-1). Minocycline was also capable of protecting the brain parenchyma from necrotic damage as evident by significantly smaller abscesses in minocycline-treated mice. In addition, minocycline exerted anti-inflammatory effects when administered as late as 3 days following <i>S. aureus</i> infection, which correlated with a significant decrease in brain abscess size. Finally, minocycline was capable of partially attenuating <i>S. aureus</i>-dependent microglial and astrocyte activation. Therefore, minocycline may afford additional therapeutic benefits extending beyond its antimicrobial activity for the treatment of central nervous system infectious diseases typified by a pathogenic inflammatory component through its ability to balance beneficial versus detrimental inflammation.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>

    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/AB.gif" alt="Analytical Biochemistry" width="150">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://www.sciencedirect.com/science/article/pii/S0003269707000498?via%3Dihub" target="_blank">Immunodetection of glycophosphatidylinositol-anchored proteins following treatment with phospholipase C</a></h2>
            <h4>Nishina KA, Supattapone S</h4>
            <h4>Anal Biochem. 2007 Apr 15;363(2):318-20</h4>
            <p align="justify">Many important regulatory proteins are attached to the extracellular leaflet of the plasma membrane by a glycophosphatidylinositol (GPI) anchor. One of the most common methods used to determine whether a protein contains a GPI anchor involves treating membranes with the enzyme phosphatidylinositol-specific phospholipase C (PI-PLC), followed by detection of the “released” candidate protein into the supernatant fraction by traditional Western blotting. Here, we report that the amounts of two representative GPI-anchored proteins, Thy-1 and the cellular prion protein (PrP<sup>C</sup>), detected by Western blotting decreased >95% following PI-PLC treatment. The diminished signals appear to be caused by failure of the PI-PLC treated proteins to adhere to PVDF or nitrocellulose membranes, and could lead to false negative results in PI-PLC release assays. Detection of PI-PLC treated PrP<sup>C</sup> molecules was successfully achieved by (1) slot blotting in the absence of SDS onto negatively charged Nylon membranes and (2) in-gel immunostaining.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>

    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/ACS.gif" alt="ACS" width="250">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://pubs.acs.org/doi/10.1021/bi061526k" target="_blank">The stoichiometry of host PrP<sup>C</sup> glycoforms modulates the efficiency of PrP<sup>Sc</sup> formation <i>in vitro</i></a></h2>
            <h4>Nishina KA, Deleault NR, Mahal SP, Baskakov I, Luhrs T, Riek R, Supattapone S</h4>
            <h4>Biochemistry. 2006 Nov 28;45(47):14129-39</h4>
            <p align="justify">A central event in the formation of infectious prions is the conformational change of a host-encoded glycoprotein, PrP<sup>C</sup>, into a pathogenic isoform, PrP<sup>Sc</sup>. However, the molecular requirements for efficient PrP conversion remain unknown. In this study, we employed the recently developed protein misfolding cyclic amplification (PMCA) and scrapie cell assay (SCA) techniques to study the role of N-linked glycosylation on prion formation <i>in vitro</i>. The results show that unglycosylated PrP<sup>C</sup> molecules are required to propagate mouse RML prions, whereas diglycosylated PrP<sup>C</sup> molecules are required to propagate hamster Sc237 prions. Furthermore, the formation of Sc237 prions is inhibited by substoichiometric levels of hamster unglycosylated PrP<sup>C</sup> molecules. Thus, interactions between different PrP<sup>C</sup> glycoforms appear to control the efficiency of prion formation in a species-specific manner.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>

        <div class="row col-md-12 col-sm-12 col-xs-12">
            <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
                <img src="Images/JBC.png" alt="JBC" width="250">
            </div>
            <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
            <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
                <h2><a href="http://www.jbc.org/content/280/29/26873.long" target="_blank">Protease-resistant prion protein amplification reconstituted with partially purified substrates and synthetic polyanions</a></h2>
                <h4>Deleault NR, Geoghegan JC, Nishina K, Kascsak R, Williamson RA, Supattapone S</h4>
                <h4>J Biol Chem. 2005 Jul 22;280(29):26873-9</h4>
                <p align="justify">Little is currently known about the biochemical mechanism by which induced prion protein (PrP) conformational change occurs during mammalian prion propagation. In this study, we describe the reconstitution of PrPres amplification <i>in vitro</i> using partially purified and synthetic components. Overnight incubation of purified PrP27–30 and PrP<sup>C</sup> molecules at a molar ratio of 1:250 yielded ∼2-fold baseline PrPres amplification. Addition of various polyanionic molecules increased the level of PrPres amplification to ∼10-fold overall. Polyanionic compounds that stimulated purified PrPres amplification to varying degrees included synthetic, homopolymeric nucleic acids such as poly(A) and poly(dT), as well as non-nucleic acid polyanions, such as heparan sulfate proteoglycan. Size fractionation experiments showed that synthetic poly(A) polymers must be >0.2 kb in length to stimulate purified PrPres amplification. Thus, one possible set of minimal components for efficient conversion of PrP molecules <i>in vitro</i> may be surprisingly simple, consisting of PrP27–30, PrPC, and a stimulatory polyanionic compound.</p>
            </div>
        </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
            
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/JBC.png" alt="JBC" width="250">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="http://www.jbc.org/content/279/39/40788.long" target="_blank">Ionic strength and transition metals control PrP<sup>Sc</sup> protease resistance and conversion-inducing activity</a></h2>
            <h4>Nishina K, Jenks S, Supattapone S</h4>
            <h4>J Biol Chem. 2004 Sep 24;279(39):40788-94</h4>
            <p align="justify">The essential component of infectious prions is a misfolded protein termed PrP<sup>Sc</sup>, which is produced by conformational change of a normal host protein, PrP<sup>C</sup>. It is currently unknown whether PrP<sup>Sc</sup> molecules exist in a unique conformation or whether they are able to undergo additional conformational changes. Under commonly used experimental conditions, PrP<sup>Sc</sup> molecules are characteristically protease-resistant and capable of inducing the conversion of PrP<sup>C</sup> molecules into new PrP<sup>Sc</sup> molecules. We describe the effects of ionic strength, copper, and zinc on the conformation-dependent protease resistance and conversion-inducing activity of PrP<sup>Sc</sup> molecules in scrapie-infected hamster brains. In the absence of divalent cations, PrP<sup>Sc</sup> molecules were >20-fold more sensitive to proteinase K digestion in low ionic strength buffers than in high ionic strength buffers. Addition of micromolar concentrations of copper or zinc ions restored the protease resistance of PrP<sup>Sc</sup> molecules under conditions of low ionic strength. These transition metals also controlled the conformation of purified truncated PrP-(27–30) molecules at low ionic strength, confirming that the N-terminal octapeptide repeat region of PrP<sup>Sc</sup> is not required for binding to copper or zinc ions. The protease-sensitive and protease-resistant conformations of PrP<sup>Sc</sup> were reversibly interchangeable, and only the protease-resistant conformation of PrP<sup>Sc</sup> induced by high ionic strength was able to induce the formation of new protease-resistant PrP (PrPres) molecules <i>in vitro</i>. These findings show that PrP<sup>Sc</sup> molecules are structurally interconvertible and that only a subset of PrP<sup>Sc</sup> conformations are able to induce the conversion of other PrP molecules.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
            
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/ACS.gif" alt="ACS" width="250">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://pubs.acs.org/doi/10.1021/bi035889l" target="_blank"><i>In vitro</i> prion protein conversion in detergent-solubilized membranes</a></h2>
            <h4>Nishina K, Deleault NR, Lucassen RW, Supattapone S</h4>
            <h4>Biochemistry. 2004 Mar 9;43(9):2613-21</h4>
            <p align="justify">A fundamental event in the pathogenesis of prion disease is the conversion of PrP<sup>C</sup>, a normal glycophosphatidyl-anchored glycoprotein, into an infectious isoform designated PrP<sup>Sc</sup>. In a modified version of the protein misfolding cyclic amplification (PMCA) technique [Saborio et al. (2001) Nature 411, 810−813], protease-resistant PrP<sup>Sc</sup>-like molecules (PrPres) can be amplified <i>in vitro</i> in a species- and strain-specific manner from crude brain homogenates, providing a biochemical model of the prion conversion reaction [Lucassen et al. (2003) Biochemistry 42, 4127−4135]. In this study, we investigated the ability of enriched membrane subsets and detergent-solubilized membrane preparations to support PrPres amplification. Membrane fractionation experiments showed that purified synaptic plasma membrane preparations enriched in PrP<sup>C</sup> but largely depleted of late endosomal and lysosomal markers were sufficient to support PrPres amplification. Detergent solubilization experiments showed that a small group of select detergents could be used to produce soluble preparations that contain PrP<sup>C</sup> and fully support PrPres amplification. The stability of PrPres amplification ability in detergent-solubilized supernatants was dependent on detergent concentration. These results lead to the surprising conclusion that membrane attachment is not required for PrP<sup>C</sup> to convert efficiently into PrPres <i>in vitro</i> and also indicate that biochemical purification of PrPres amplification factors from brain homogenates is a feasible approach.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
    
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/JNC.gif" alt="JNC" width="150">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://onlinelibrary.wiley.com/doi/full/10.1046/j.1471-4159.2003.01819.x?sid=nlm%3Apubmed" target="_blank">Post-transcriptional suppression of pathogenic prion protein expression in <i>Drosophila</i> neurons</a></h2>
            <h4>Deleault NR, Dolph PJ, Feany MB, Cook ME, Nishina K, Harris DA, Supattapone S</h4>
            <h4>J Neurochem. 2003 Jun;85(6):1614-23</h4>
            <p align="justify">A wealth of evidence supports the view that conformational change of the prion protein, PrP<sup>C</sup>, into a pathogenic isoform, PrP<sup>Sc</sup>, is the hallmark of sporadic, infectious, and inherited forms of prion disease. Although the central role played by PrP<sup>Sc</sup> in the pathogenesis of prion disease is appreciated, the cellular mechanisms that recognize PrP<sup>Sc</sup> and modulate its production, clearance, and neural toxicity have not been elucidated. To address these questions, we used a tissue-specific expression system to express wild-type and disease-associated PrP molecules heterologously in <i>Drosophila melanogaster</i>. Our results indicate that <i>Drosophila</i> brain possesses a specific and saturable mechanism that suppresses the accumulation of PG14, a disease-associated insertional PrP mutant. We also found that wild-type PrP molecules are maintained in a detergent-soluble conformation throughout life in <i>Drosophila</i> brain neurons, whereas they become detergent-insoluble in retinal cells as flies age. PG14 protein expression in <i>Drosophila</i> eye did not cause retinal pathology. Our work reveals the presence of mechanisms in neurons that specifically counterbalance the production of misfolded PrP conformations, and provides an opportunity to study these processes in a model organism amenable to genetic analysis.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
        
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/ACS.gif" alt="ACS" width="250">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://pubs.acs.org/doi/10.1021/bi027218d" target="_blank"><i>In Vitro</i> Amplification of Protease-Resistant Prion Protein Requires Free Sulfhydryl Groups</a></h2>
            <h4>Ralf Lucassen, Koren Nishina, Surachai Supattapone</h4>
            <h4>Biochemistry, 2003, 42 (14), pp 4127–4135</h4>
            <p align="justify">Prions, the infectious agents of transmissible spongiform encephalopathies, are composed primarily of a misfolded protein designated PrP<sup>Sc</sup>. Prion-infected neurons generate PrP<sup>Sc</sup> from a host glycoprotein designated PrP<sup>C</sup> through a process of induced conformational change, but the molecular mechanism by which PrP<sup>C</sup> undergoes conformational change into PrP<sup>Sc</sup> remains unknown. We employed an <i>in vitro</i> PrP<sup>Sc</sup> amplification technique adapted from protein misfolding cyclic amplification (PMCA) to investigate the mechanism of prion-induced protein conformational change. Using this technique, PrP<sup>Sc</sup> from diluted scrapie-infected brain homogenate can be amplified >10-fold without sonication when mixed with normal brain homogenate under nondenaturing conditions. PrP<sup>Sc</sup> amplification <i>in vitro</i> exhibits species and strain specificity, depends on both time and temperature, only requires membrane-bound components, and does not require divalent cations. <i>In vitro</i> amplification of Syrian hamster Sc237 PrP<sup>Sc</sup> displays an optimum pH of ∼7, whereas amplification of CD-1 mouse RML PrP<sup>Sc</sup> is optimized at pH ∼6. The thiolate-specific alkylating agent N-ethylmaleimide (NEM) as well as the reversible thiol-specific blockers p-hydroxymercuribenzoic acid (PHMB) and mersalyl acid inhibited PrP<sup>Sc</sup> amplification <i>in vitro</i>, indicating that the conformational change from PrP<sup>C</sup> to PrP<sup>Sc</sup> requires a thiol-containing factor. Our data provide the first evidence that a reactive chemical group plays an essential role in the conformational change from PrP<sup>C</sup> to PrP<sup>Sc</sup>.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>
    
    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/BCP.gif" alt="Biochemical Pharmacology" width="150">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://www.sciencedirect.com/science/article/abs/pii/S0006295202008742?via%3Dihub" target="_blank">Pharmacological approaches to prion research</a></h2>
            <h4>Surachai Supattapone, Koren Nishina, Judy R. Rees</h4>
            <h4>Volume 63, Issue 8, 15 April 2002, Pages 1383-1388</h4>
            <p align="justify">The “protein-only” mechanism by which infectious agents of prion diseases such as Creutzfeldt–Jakob disease and bovine spongiform encephalopathy replicate remains undetermined. The identification of several distinct classes of prion inhibitors has created an opportunity to investigate the mechanism of prion formation using pharmacological tools. These new inhibitors include substituted tricyclic derivatives, tetrapyrrole compounds, cysteine protease inhibitors, branched polyamines, and specific antibodies. Each inhibitor class contains at least one active compound that inhibits prion propagation in cell culture at sub-micromolar concentrations and several structurally related, inactive compounds. Work with branched polyamines and specific antibodies has already provided insight into the kinetics and cell biology of endogenous prion clearance mechanisms. Other anti-prion compounds do not appear to bind directly to the prion protein. Detailed investigation of the mechanism of drug action of these compounds may lead to the identification of novel prion propagation factors.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <hr>
    </div>

    <div class="row col-md-12 col-sm-12 col-xs-12">
        <div class="paper_image col-md-3 col-sm-3 col-xs-3 container" style="margin-left: 50px;">
            <img src="Images/AEM.png" alt="Applied and Environmental Microbiology" width="300">
        </div>
        <div class="row col-md-1 col-sm-1 col-xs-1 container"> </div>
        <div class="paper_abstract col-md-7 col-sm-7 col-xs-7">
            <h2><a href="https://aem.asm.org/content/66/10/4440.long" target="_blank">Identification and analysis of the polyhydroxyalkanoate-specific beta-ketothiolase and acetoacetyl coenzyme A reductase genes in the cyanobacterium <i>Synechocystis</i> sp. strain PCC6803</a></h2>
            <h4>Taroncher-Oldenburg G, Nishina K, Stephanopoulos G</h4>
            <h4>Appl Environ Microbiol. 2000 Oct;66(10):4440-8</h4>
            <p align="justify"><i>Synechocystis</i> sp. strain PCC6803 possesses a polyhydroxyalkanoate (PHA)-specific beta-ketothiolase encoded by phaA<sub>Syn</sub> and an acetoacetyl-coenzyme A (CoA) reductase encoded by phaB<sub>Syn</sub>. A similarity search of the entire <i>Synechocystis</i> genome sequence identified a cluster of two putative open reading frames (ORFs) for these genes, slr1993 and slr1994. Sequence analysis showed that the ORFs encode proteins having 409 and 240 amino acids, respectively. The two ORFs are colinear and most probably coexpressed, as revealed by sequence analysis of the promoter regions. Heterologous transformation of <i>Escherichia coli</i> with the two genes and the PHA synthase of <i>Synechocystis</i> resulted in accumulation of PHAs that accounted for up to 12.3% of the cell dry weight under high-glucose growth conditions. Targeted disruption of the above gene cluster in <i>Synechocystis</i> eliminated the accumulation of PHAs. ORFs slr1993 and slr1994 thus encode the PHA-specific beta-ketothiolase and acetoacetyl-CoA reductase of <i>Synechocystis</i> and, together with the recently characterized PHA synthase genes in this organism (S. Hein, H. Tran, and A. Steinbüchel, Arch. Microbiol. 170:162-170, 1998), form the first complete PHA biosynthesis pathway known in cyanobacteria. Sequence alignment of all known short-chain-length PHA-specific acetoacetyl-CoA reductases also suggests an extended signature sequence, VTGXXXGIG, for this group of proteins. Phylogenetic analysis further places the origin of phaA<sub>Syn</sub> and phaB<sub>Syn</sub> in the gamma subdivision of the division Proteobacteria.</p>
        </div>
    </div>
    <div class="row col-md-12 col-sm-12 col-xs-12 paper_box">
        <br />
        <br />
    </div>

    

    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>


</body>



</html>